internal/errors/error.go

```diff
@@ -1,12 +1,5 @@
 package errors
 
-var (
-	badRequest       = newError("bad request")
-	notFound         = newError("not found")
-	positionIsExists = newError("position is exists")
-	employeeIsExists = newError("employee is exists")
-)
-
 type Errors struct {
 	description string
 }
```

Comment:
I don't understand the purpose of the package in it current state.
What's the reason to have a separate  package which basically does the same as `errors.New()`?